### DISCUSSION TITLE
Can't find model 'en_core_web_lg'

### DISCUSSION THREAD

**Discussion by robnev1** [explosion/spaCy#12397] [Help: Installation]
first post newbie etc. I am trying to do topic modelling and have downloaded spacy but when trying to use the en_core_web_lg i receive the following error message:

OSError: [E050] Can't find model 'en_core_web_lg'. It doesn't seem to be a Python package or a valid path to a data directory.

i have tried to use the anaconda powershell with this code: pip install -U spacy python -m spacy download en_core_web_sm

but i get an error AttributeError: module 'click.utils' has no attribute '_expand_args'

i have also tried conda install -c conda-forge spacy-model-en_core_web_lg in powershell but this errors and i have also tried from kaggle  import spacy.cli
spacy.cli.download("en_core_web_lg")
nlp = spacy.load("en_core_web_lg")

but this also errors 'HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /explosion/spacy-models/master/compatibility.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)')'

any help appreciated

**Comment by thomashacker** [comment]
Hey, thanks for your post!

Have you tried to install spaCy from a new virtual environment? 
The SSL errors seem to be caused by your network configuration or firewall.
